openapi/paths/kyc-settings.yaml

```
@@ -1,3 +1,33 @@
+get:
+  x-products:
+    - Core
+  tags:
+    - KYC Documents
+  summary: Get KYC settings
+  operationId: GetKycSettings
+  x-sdk-operation-name: getKycSettings
+  description: >
+    Retrieve KYC settings.
+  requestBody:
+    content:
+      application/json:
+        schema:
+          $ref: ../components/schemas/KycSettings.yaml
+    description: The settings for KYC analysis.
+    required: true
+  responses:
+    '200':
+      description: KYC settings were retrieved successfully.
```

nitpick: avoid past tense
```suggestion
      description: KYC settings successfully retrieved .
```